build/gh-pages-app.js

```diff
@@ -147,33 +147,30 @@ class GhPagesApp {
 
   /**
    * @param {string} input
-   * @param {import('rollup').Plugin[]=} plugins
+   * @param {esbuild.Plugin[]=} plugins
    * @return {Promise<string>}
    */
-  async _rollupSource(input, plugins) {
-    plugins = plugins || [
-      rollupPlugins.nodeResolve(),
-      rollupPlugins.commonjs(),
-    ];
-    if (!process.env.DEBUG) plugins.push(rollupPlugins.terser());
-    const bundle = await rollup({
-      preserveEntrySignatures: 'strict',
-      input,
+  async _esbuildSource(input, plugins) {
+    const result = await esbuild.build({
+      entryPoints: [input],
+      write: false,
+      outdir: fs.mkdtempSync(path.join(os.tmpdir(), 'gh-pages-app-')),
+      format: 'esm',
+      bundle: true,
+      splitting: true,
+      minify: !process.env.DEBUG,
```

Comment:
I thought we didn't want to use esbuild minify yet, or was that just for dt/lr bundles?

Reply:
Gotcha but also https://github.com/GoogleChrome/lighthouse/pull/15239/files#r1263986583